interface.h

```diff
@@ -1,29 +1,32 @@
-// ----------------------------------------------------------------------------
-// multipass interface for interacting with hardware in a platform agnostic way
-// each supported platform will have its own implementation
-//
-// any apps using multipass should only talk to hardware using functions 
-// defined here
-// 
-// needs control.h for the data types used for preset management
-// ----------------------------------------------------------------------------
+/**
+ * @file interface.h
+ * 
+ * @brief Multipass interface for interacting with hardware in a platform
+ *        agnostic way each supported platform will have its own implementation.
+ * 
+ *        Any apps using Multipass should only talk to hardware using functions 
+ *        defined here.
+ * 
+ *        Needs `control.h` for the data types used for preset management
+ * 
+ */
 
 #pragma once
 
 #include "control.h"
 
-#define MAX_LEVEL 16383
+#define MAX_LEVEL 16383 /**< Maximum CV level value */
 
 
 // ----------------------------------------------------------------------------
 // voice mapping
 
-#define VOICE_CV_GATE     0
-#define VOICE_ER301       1
-#define VOICE_JF          2
-#define VOICE_TXO_NOTE    3
-#define VOICE_TXO_CV_GATE 4
-#define MAX_DEVICE_COUNT  5
+#define VOICE_CV_GATE     0 /**< Voice mapping identifier for internal Monome hardware CV/Gate output pair */
+#define VOICE_ER301       1 /**< Voice mapping identifier for the [Orthogonal Devices ER-301](http://www.orthogonaldevices.com/er-301) */
+#define VOICE_JF          2 /**< Voice mapping identifier for the [Mannequins Just Friends](https://www.whimsicalraps.com/products/just-friends) */
+#define VOICE_TXO_NOTE    3 /**< Voice mapping identifier for the [TELEXo](https://github.com/bpcmusic/telex) in sound mode */
```

Comment:
since the makers are mentioned for other modules, could you change it to `[bpcmusic TELEXo]`?